src/Network/Network/VirtualNetwork/Subnet/ResetAzureVirtualNetworkSubnetPolicyCommand.cs

```diff
@@ -0,0 +1,107 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Management.Network.Models;
+using System;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.Reset, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "VirtualNetworkSubnetPolicy", SupportsShouldProcess = true, DefaultParameterSetName = "SetByResourceId"), OutputType(typeof(PSSubnet))]
+    public class ResetAzureVirtualNetworkSubnetPolicyCommand : VirtualNetworkBaseCmdlet
+    {
+
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = "DefaultParameterSet",
+            HelpMessage = "The name of the subnet")]
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = "SetByResourceId",
+            HelpMessage = "The name of the subnet")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
```

Comment:
This does not implement the parameter sets as suggestedin the cmdlet review.  Please complete the cmdlet review

Reply:
I have implemented all the parameter sets recommended for initialize. This is in parity with initialize command. I have mentioned the same in design review as well.